utils/postgres_writer.py

```diff
@@ -11,17 +12,16 @@ def insert_postgres_table_if_rows_not_exist(
     """
     Inserts non-duplicate rows into PostreSQL table in batches.
 
-    Parameters:
-    - dataframe (pd.DataFrame): The DataFrame to be written.
-    - table_name (str): The name of the PostgreSQL table.
-    - engine (sqlalchemy.engine.base.Engine): SQLAlchemy engine for database connection.
-    - unique_key_columns (list): Columns that will prevent data insert on conflict.
+    Args:
+        dataframe (pd.DataFrame): The DataFrame to be written.
+        table_name (str): The name of the PostgreSQL table.
+        engine (sqlalchemy.engine.base.Engine): SQLAlchemy engine for database connection.
+        unique_key_columns (list): Columns that will prevent data insert on conflict.
 
     Returns:
-    - None (or Error)
+        None (or Error)
```

Comment:
Can you be a bit more specific as it is a None when successful and the Error string if not?